account_invoice_show_currency_rate/tests/test_account_move.py

```diff
@@ -11,6 +11,7 @@ def setUpClass(cls):
         super().setUpClass()
         cls.currency_usd = cls.env.ref("base.USD")
         cls.currency_eur = cls.env.ref("base.EUR")
+        cls.env.ref("base.main_company").write({"currency_id": cls.currency_usd.id})
```

Comment:
This change is forbidden by a constraint in Odoo itself, so this is not the correct thing. Several options:

1. Create a new company, which can be too much.
2. Change the currency of the company by SQL as a hackish woraround.
3. Have the secondary currency in a variable depending on the company currency, and do the rest of the tests according it.

Reply:
Ok, I have made the changes according to option 3,